content/blog/lifehacking/2017-12-01_freedom_and_productivity.md

```diff
@@ -0,0 +1,199 @@
+with our principles and real objectives. Thus it is about freedom, that is, the
+way to express what you truly desire to do and then to do it with purpose. Or
+in the words of French writer and journalist Françoise Giroud: happiness is
+doing what you want, and wanting what you do. I think other people call this
+autonomy or even integrity; I hope you get the idea.
+
+In particular, I don't want others to preconceive life choices for myself. Even
+if they have theoritized the topic and proven with objective metrics it worked
+for them. So here is my meta-productivity advice: to try different methods,
+and to make your own solutions out of them, to choose what works according to
+the context. Even if I need to spend a lot of time on thinking things through
+first, before applying my own crafted solutions, it is such a lower entry cost
+than doing erratic stuff without purpose and realize I lost my time. Of course,
+I might not find the best solution on the first try, but that's alright. It
+might mean I switch routines from one week to the other, or I don't have a
+consistent agenda; that is okay. I can experiment with many possible ways to do
+one thing, according to my current context: my state of mind, geographical
+location, needs, expectations, levels of energy, unexpected events popping up
+throughout the day, political context, really anything. As long as everything I
+do is conscious and works for me and makes me feel more fulfilled, I'll do the
+tradeoff of not having perfect control and a predefined answer to everything.
+
+What some call the review process matters here: one has to make sure they're
+doing their own right thing, in terms of process and goals, at periodic
+intervals. Otherwise, there's a risk to get drawn in the froth of the days,
+lose sight on what truly matters to us and get lost. During this process, it is
+important to feel like an external observer to yourself, so as to avoid falling
+into judgement and then feel overwhelmed or unhappy with yourself. You can for
+instance consider you're a scientific researcher that just tries to assess
+hypothesis and confront them with the reality. Or anything else really; as long
+as you keep it fun!
+
+Also, blocking time for rest and non-work activities is important; neuroscience
+has proven that the brain mostly works under two modes, a focused mode (when we
+consciously think about or work on a given problem) and a diffused mode (that
+gets triggered whenever we rest, do errands). Getting stuck in one of the twos
+is the best way to miss better solutions or do subefficient work. It seems
+probably obvious to most that the former is needed, but a lot of people tend to
+underestimate the usefulness of the latter. A lot of background processing
+happens in this mode, and interesting solutions can emerge out of this mind
+state: this might have happened to you through dreams, reveries, showers or
+workout breakthroughs, for instance.
```

Comment:
Great point. I think Stephanie Hurlburt made an amazing tweet about this, let me find it... (20 seconds later) https://twitter.com/sehurlburt/status/891763445586112514 (thank you Google! I never thought "sehurlburt 4 hours a day" would find me this tweet again so easily)

Reply:
Her second tweet is also amazing: https://twitter.com/sehurlburt/status/891763863091257344

Reply:
Nice tweets! It is comforting to read it, as I have been telling people I work 20 hours per week too. Not that I feel guilty for it, but still, it's nice to hear that I'm not alone. :)